### ISSUE TITLE
Clink 1.0.0.a1 - `clink set history_io 1` Error

### ISSUE THREAD

**Issue by daxgames** [mridgers/clink#520]
Works perfectly with Clink 0.4.9.

Fails with Clink 1.0.0.a1.
```
C:\>clink set history_io 1
ERROR: Setting 'history_io' not found.
```

1. Why was this removed.
2. Can it be added back.

**Comment by chrisant996** [comment]
You're right, the setting was removed -- it always loads and saves history.
I'll add a setting to allow turning off persistent history in the [chrisant996/clink](https://github.com/chrisant996/clink) fork.

**Comment by daxgames** [comment]
@chrisant996 thanks.  Looking forward to a release and hoping it will work as a replacement for Clink 0.4.9 in Cmder that will also work with latest Windows insider builds.

**Comment by chrisant996** [comment]
> Looking forward to a release and hoping it will work as a replacement for Clink 0.4.9 in Cmder 

Yes, that is an overt goal.  Getting it working standalone is the first step.

> that will also work with latest Windows insider builds.

I don't currently have a machine with a Windows insider build.  In the future, I expect to set one up.

**Comment by daxgames** [comment]
Yeah we tried to look at the 1.x.x alphas but they were very alpha.

**Comment by chrisant996** [comment]
> Yeah we tried to look at the 1.x.x alphas but they were very alpha.

Yep.  The structure of the code is good.  There are a lot of bugs, and sometimes they take hours to track down.  But the majority of them have turned out to be little mistakes or typos, and the overall "feel" is a quality code base that was abandoned shortly before it would have reached beta quality level.  I'm grateful to Martin for releasing the code in alpha quality (vs not sharing it!), and I'm happy to help take it across the goal line.

The two things I see that will be blocking issues for cmder are:
- The `-s` flag.
- The lua argmatcher syntax.  My goal is to add a compatibility shim so that existing scripts continue to work with the old syntax.  Or if that eludes me then I'll seek help and/or revert to the old syntax and then work on adding the new capabilities into the old syntax.

Please feel free to open issues in the chrisant996/clink repo about things the cmder team sees as potential blocking issues, and etc.
